### ISSUE TITLE
Install fails on OpenBSD

### ISSUE THREAD

**Issue by vext01** [trezor/cython-hidapi#20]
```
$ pip2.7 install --user hidapi
You are using pip version 7.1.0, however version 7.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting hidapi
  Using cached hidapi-0.7.99.post8.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-zHt5mZ/hidapi/setup.py", line 79, in <module>
        ext_modules = modules,
    NameError: name 'modules' is not defined
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-zHt5mZ/hidapi
```

Looking at the code, I can immediately see why:

```
if sys.platform.startswith('linux'):
...
if sys.platform.startswith('darwin'):
...
if sys.platform.startswith('win'):
...
ext_modules = modules,
```

On OpenBSD 5 `sys.platform` is `openbsd5`, therefore `modules` never gets defined.

I'm not sure what the correct dependencies would be for OpenBSD, but maybe similar to linux, albeit without udev.

Thanks

**Comment by prusnak** [comment]
Could you please clone the repository, try modifying the setup.py file the way you think it will work and running `python setup.py install` ? Once you have the solution you can post it here or create a pull request.

**Comment by prusnak** [comment]
Closed in aa0de786f29b884f9a13bf224d91773635a6ed19
